lib/pure/oids.nim

```diff
@@ -51,45 +48,60 @@ proc parseOid*(str: cstring): Oid =
     bytes[i] = chr((hexbyte(str[2 * i]) shl 4) or hexbyte(str[2 * i + 1]))
     inc(i)
 
-proc oidToString*(oid: Oid, str: cstring) =
-  ## Converts an oid to `str` which must have space allocated for 25 elements.
+template toStringImpl[T: string | cstring](result: var T, oid: Oid) =
+  ## Converts an oid to `str`.
```

Comment:
```suggestion
  ## stringify `oid`
```

(`str` doesn't refer to anything anymore)
(and maintains DRY-ness; we shouldn't repeat in docs what's implied in declaration)